Feature/ae 1770 error messages

muep opened the pull request.

muep committed:
AE-1770 Announce the text of Alert

muep committed:
Add some explanatory comments in validation.js

Juholei commented on src/utils/validation.js:
```diff
@@ -280,6 +286,11 @@ const validate = (validators, value) =>
  * @sig Array [Validator] -> a -> Either [(Translate -> string),a]
+ *
+ * This function checks a single value from the data model
+ * against an array of validators. The value is pasesd through
```
```suggestion
 * against an array of validators. The value is passed through
```

Juholei reviewed (approved):
Eikö meillä siis ole mitään html-sivupohjaa tällä hetkellä vai miten tuo oikein toimii?

muep committed:
AE-1770 Use aria-invalid in inputs

This makes it easier to spot inputs needing attention while
the user navigates through a form

muep committed:
AE-1770 Add a description label with validation error

muep committed:
Fix typo in src/utils/validation.js

Co-authored-by: Juho Leinonen <juho.leinonen@solita.fi>

muep committed:
Reformat

muep commented:
> Eikö meillä siis ole mitään html-sivupohjaa tällä hetkellä vai miten tuo oikein toimii?

Näytti että ei ollut ennestään meidän määräämää pohjaa. Varmaan sitten webpackista tuli joku oletuspohja?
